### PR TITLE
Fix Examples for Proposer Selection Specs

### LINKED ISSUE #9346: Change examples in proposer selection sepcification.
# Protocol Change Proposal

## Summary
Change examples in Tendermint specifications in [proposer selection ](https://github.com/tendermint/tendermint/blob/v0.34.x/spec/consensus/proposer-selection.md)for consistency. 

## Problem Definition
What problems may be addressed by introducing this change?
The examples have, in my opinion, some minor mistakes. 
What benefits does Tendermint stand to gain by including this change?
Specs are easier to understand. 
Are there any disadvantages of including this change? --> I don't think so

### Validator Removal 
The priority should be from -3 to 3 
After __new step__, the priority from p1 should increase by 1 from 2 to 3 as it has VP 1

### New Validator 
The new proposer priority of p3 should be set to -13 and not -4 in **Alg step** 

### Proposer Priority Change
After you add p2 and run a selection, the priority of p1 is 45k and of p2 is -45k. The signs a flipped in the first Table. 

PS: I failed to do the rebase in #9326, so I am abandoning it and trying to do it right from the start, starting with an issue.

### PR DESCRIPTION
**Pull request by nashqueue**
This PR resolves #9346

### CODE REVIEW

**Comment by sergio-mena** [comment]
I will try to take a look next week if that's OK, after we're done with the QA process for v0.37.x

File: spec/consensus/proposer-selection.md [hunk 1]
```diff
@@ -136,13 +136,13 @@ Let's assume that after the last run the proposer priorities were as shown in fi
...
-| __new step__   |    | p3 |    |   |    | p1 |   | A(i) -= avg, avg = -1 |
-|                |    |    |    |   | p3 | p1 |   | A(i)+=VP(i)           |
-|                |    |    | p1 |   | p3 |    |   | A(p1)-= P             |
+| Priority   Run | -3 | -2 | -1 | 0 | 1  | 2  | 3  | Comment               |
+|----------------|----|----|----|---|----|----|----|-----------------------|
+| last run       | p3 |    |    |   | p1 | p2 |    | __remove p2__         |
+| nextrun        |    |    |    |   |    |    |    |                       |
+| __new step__   |    | p3 |    |   |    | p1 |    | A(i) -= avg, avg = -1 |
+|                |    |    |    |   | p3 |    | p1 | A(i)+=VP(i)           |
+|                |    |    | p1 |   | p3 |    |    | A(p1)-= P             |
```
Comment (adizere) [inline spec/consensus/proposer-selection.md:145]: Looks better, thanks for this correction!

File: spec/consensus/proposer-selection.md [hunk 2]
```diff
@@ -200,7 +200,7 @@ In the next run, p3 will still be ahead in the queue, elected as proposer and mo
 | Priority   Run | -13 | -9 | -5 | -2 | -1 | 0 | 1 | 2  | 5  | 6  | 7  | Alg step              |
 |----------------|-----|----|----|----|----|---|---|----|----|----|----|-----------------------|
 | last run       |     |    |    | p2 |    |   |   | p1 |    |    |    | __add p3__            |
-|                | p3  |    |    | p2 |    |   |   | p1 |    |    |    | A(p3) = -4            |
+|                | p3  |    |    | p2 |    |   |   | p1 |    |    |    | A(p3) = -13           |
```
Comment (adizere) [inline spec/consensus/proposer-selection.md:203]: Good catch.

File: spec/consensus/proposer-selection.md [hunk 3]
```diff
@@ -215,7 +215,7 @@ Validator | p1   | p2   | Comment
 ----------|------|------|------------------
 VP        | 80k  | 10   |
 A         | 0    | -90k | __added p2__
-A         | -45k | 45k  | __run selection__
+A         | 45k  | -45k | __run selection__
```
Comment (adizere) [inline spec/consensus/proposer-selection.md:218]: Good catch.

**Review by adizere — APPROVED** [review]
Read the spec with an emphasis on the modified parts. The corrections make sense, thank you for your contribution!

The spec overall, however, could use some maintenance: it has a few open todos and remarks about needing more fleshing-out, but that's outside the scope here.

**Review by adizere — APPROVED** [review]

**Comment by nashqueue** [comment]
Any Update? :) @sergio-mena @adizere @lasarojc

**Comment by adizere** [comment]
> Any Update? :) 

Thanks for the reminder, merged!

**Pull request merged**